source/_integrations/thread.markdown

```diff
@@ -11,5 +11,12 @@ ha_integration_type: service
 ha_config_flow: true
 ---
 
-The Thread integration is currently a placeholder, but will manage Thread network credentials in a future release.
-The integration is automatically setup when the "Silicon Labs Multiprotocol" add-on is installed.
+The Thread integration helps you track the different Thread networks in your home, the credentials that they use and helps you align them. It is currently a work in progress.
```

Comment:
The Thread integration helps you track the different Thread networks in your home and helps you manage their credentials.